sixtyfps_runtime/rendering_backends/qt/qt_window.rs

```diff
@@ -1277,6 +1277,15 @@ impl PlatformWindow for QtWindow {
             }
             widget_ptr->setWindowTitle(title);
             auto pal = widget_ptr->palette();
+
+            // If the background color is the same as what NativeStyleMetrics supplied from QGuiApplication::palette().color(QPalette::Window),
+            // then the setColor (implicitly setBrush) call will not detach the palette. However it will set the resolveMask, which due to the
+            // lack of a detach changes QGuiApplicationPrivate::app_pal's resolve mask and thus breaks future theme based palette changes.
+            // Therefore we force a detach
+            {
+                pal.setResolveMask(~pal.resolveMask());
+                pal.setResolveMask(~pal.resolveMask());
```

Comment:
why do you call it twice?
maybe you want to be explicit and call pal.detach() ?

Reply:
detach() is a private function, so that won't compile with MSVC I think :(